docs/community/documentation-style-guide.rst

```diff
@@ -0,0 +1,377 @@
+
+- start conditional phrases with the conditional part, so that the reader
+  may skip the rest if it does not apply
+
+=======================================  =====================================
+Do                                       Don't
+=======================================  =====================================
+If you delete a record, you will not...  You will not...if you delete a record
+=======================================  =====================================
+
+- avoid using modifiers that do not add value, for example, starting a
+  sentence with "There are/is"
+
+=======================================  =====================================
+Do                                       Don't
+=======================================  =====================================
+To start...                              There are several ways to start...
+Click the button...                      Simply click the button...
+To start, use...                         It's very easy to start, use...
+Click the next...                        Please click the next...
+=======================================  =====================================
+
+- first mention the objective of an action, and only afterwards its
+  description, for example "To do X, click Y..."
+
+=======================================  =====================================
+Do                                       Don't
+=======================================  =====================================
+To do X, click Y                         Click Y, to do X
+=======================================  =====================================
+
+- do repeat nouns instead of using pronouns, in cases where there are more
+  than one subjects, to avoid ambiguity:
+
+======================================================================   ===============================================================
+Do                                                                       Don't
+======================================================================   ===============================================================
+if you create a file in a specific bucket, make sure the file is named   if you create a file in a specific bucket, make sure it's named
+======================================================================   ===============================================================
+
+Technical writing style guidelines
+----------------------------------
+Keep in mind that the person reading your documentation may have no
+knowledge of what the module does.
+Always give a general, brief overview of any new term and explain its
+function and purpose.
+Remember to be consistent, do not use different words for the same
+concept throughout the text.
+
+Highlighting and formatting
+---------------------------
+Formatting guidelines:
+
+- use camel case or snake case where appropriate and do not split
+  object/function names into separate words.
```

Comment:
could you either remove the dot from the end of the line or add it to the other bullet points? I'd go for the first

Reply:
Good spot! Will do